css/theme.css

```diff
@@ -49,18 +49,24 @@ body {
 }
 
 header {
+  width: auto;
+  max-width: 1000px;
+  padding: 200px;
```

Comment:
this is currently causing a big gap between the header content and the main body